src/api/cart.js

```diff
@@ -2,23 +2,57 @@ import Base from './base';
 import Hooks from '../hooks';
 
 export default class extends Base {
+    /**
+     * Get a collection of Carts. For now, this will only return an array of a single cart as multiple carts per session
+     * are not currently supported.
+     *
+     * @param options
+     * @param {Function} callback
+     */
+    getCarts(options = {}, callback) {
```

Comment:
What is the right use case for this ?

Reply:
I thought this was the best way to represent the array of carts returned by `/api/storefront/cart`. The reason that endpoint returns an array is because customers might be able to have more than one shopping cart in the future. So this is just future-proofing.

Reply:
Also just to confirm we have `/api/storefront/carts` endpoint right ? This one is plural.

Reply:
Yes, they both work, and they both return an array.